modules/dynamic_backends/main.tf

```diff
@@ -111,11 +111,12 @@ resource "google_compute_backend_service" "default" {
   description                     = lookup(each.value, "description", null)
   connection_draining_timeout_sec = lookup(each.value, "connection_draining_timeout_sec", null)
   enable_cdn                      = lookup(each.value, "enable_cdn", false)
-  security_policy                 = lookup(each.value, "security_policy", null) == null ? null : var.security_policy
   health_checks                   = lookup(each.value, "health_check", null) == null ? null : [google_compute_health_check.default[each.key].self_link]
   session_affinity                = lookup(each.value, "session_affinity", null)
   affinity_cookie_ttl_sec         = lookup(each.value, "affinity_cookie_ttl_sec", null)
   custom_request_headers          = lookup(each.value, "custom_request_headers", [])
+  # To achieve a null backend security_policy, set each.value.security_policy to "" (empty string).
+  security_policy = lookup(each.value, "security_policy", null) == null ? var.security_policy : each.value.security_policy
```

Comment:
Shouldn't we actually check for the "" value?

Reply:
So I manually tested it with this code and it appears that `""` is interpreted as `null` so we don't have to explicitly set `null` via another ternary.

However, this may be a terraform thing and may be limited to certain versions of terraform, so being explicit may be safer.

Reply:
Yeah, I'd rather explicitly check for the value.